### PR TITLE
Binary tree to bst

### LINKED ISSUE #250: Algorithm to convert Binary Tree into Binary Search Tree
Given a Binary Tree, convert it to a Binary Search Tree. The conversion must be done in such a way that keeps the original structure of Binary Tree. 
I can code it in Java, Python and C++.
@Kumar-laxmi Please assign this to me under SWoC'23

### PR DESCRIPTION
**Pull request by Aaditya1612**
## 🛠️ Issue (Number)

issue no #250 

## ✔️ Check List (Check all the applicable boxes) 

- [✔️ ] My code follows the code style of this project.
- [ ✔️] This PR does not contain plagiarized content.
- [ ✔️] The title of my pull request is a short description of the requested changes.

## 📷 Screenshots

**BinaryTreeToBST.cpp Output**

image

**BinaryTreeToBST.java Output**

image

**BinaryTreeToBST.py Output**

image

**BinaryTreeToBST.c Outptu**

image

### CODE REVIEW

**Comment by Aaditya1612** [comment]
@Kumar-laxmi  I have resolved the issue. 
Review Requested.

File: C++/Trees/BinaryTreeToBST.cpp [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#include <bits/stdc++.h>
+using namespace std;
+
```
[the remark this answers is not in the document]
Comment (Aaditya1612) [reply C++/Trees/BinaryTreeToBST.cpp:3]: OK

[the remark this answers is not in the document]
Comment (Aaditya1612) [reply C++/Trees/BinaryTreeToBST.cpp:3]: @zugzwang03 I have updated the code as per requirements.

**Review by zugzwang03 — APPROVED** [review]
LGTM!

**Review by Sanjanabharadwaj25 — APPROVED** [review]

**Review by 072arushi — APPROVED** [review]
approved

**Review by Kumar-laxmi — APPROVED** [review]

**Pull request merged**